app/views/checkout/_details.html.haml

```diff
@@ -95,7 +95,7 @@
         %div.checkout-input{"data-shippingmethod-target": "shippingMethodDescription", "data-shippingmethodid": shipping_method.id , style: "display: #{ship_method_is_selected ? 'block' : 'none'}" }
           #distributor_address.panel
             - if shipping_method.description.present?
-              %span #{shipping_method.description}
+              %span #{simple_format(shipping_method.description)}
```

Comment:
Nice simple solution. The `simple_format` doesn't sanitise HTML though which would otherwise be done automatically. So your change would introduce a security issue by allowing people to insert malicious HTML. The docs recommend using the `h` helper which is an alias for `html_escape`. Maybe let's make that clear here:

```rb
simple_format(html_escape(shipping_method.description))
```

And we can write this a bit clearer in HAML:

```suggestion
              %span
                = simple_format(html_escape(shipping_method.description))
```

Actually, the `simple_format` call is adding a `<p>` tag around the description. That shouldn't be within a span. So maybe we should leave it out:

```suggestion
              = simple_format(html_escape(shipping_method.description))
```

Reply:
Went to read the doc again in the original issue that i posted (https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#M002285) but it says that simple_format would not sanitize the html presenting a security risk like you said. Thought otherwise. 

The comment is so good and clear, learned a bit by reading it, thank you.

(Will add the changes in another time, cause in portugal it's quite a bit late.)